File: Axolotl/Source/Axolotl/BAModel/Model/GlobalModels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using BAModel.Model;

namespace BAModel.Model
{
    class GlobalModels
    {
    }

    public class GetCompanyList_drp_response
    {
        public List<Dropdown_Model> records { get; set; }
        public ResultStatus result { get; set; }

        public GetCompanyList_drp_response()
        {
            result = new ResultStatus();
            records = new List<Dropdown_Model>();
        }
    }
}

File: Axolotl/Source/Axolotl/Axolotl/Axolotl_Web/src/app/AppModel/Global_Models.ts
import { ResultStatus, Dropdown_Model } from '../AppModel/appmodel_Model'

export class GetCompanyList_drp_response {
  records: Dropdown_Model[];
  result: ResultStatus;

  public constructor()
{
    this.result = new ResultStatus();
    this.records = new Array<Dropdown_Model>();
}
    }
